src/driver.rs

```diff
@@ -68,6 +68,7 @@ pub enum TelemetryEvent {
 pub struct TreeStats {
     pub time: Duration,
     pub items: usize,
+    pub deletions: usize,
```

Comment:
`merged_deletions` is gone, but we can still report the number of local and remote deletions in shutdown hangs.